src/Database/Log/LoggingStatement.php

```diff
@@ -75,6 +77,21 @@ public function execute(?array $params = null): bool
             $result = parent::execute($params);
             $this->loggedQuery->took = (int)round((microtime(true) - $this->startTime) * 1000, 0);
         } catch (Exception $e) {
+            $this->loggedQuery->error = $e;
+            $this->_log();
+
+            if (Configure::read('Error.convertStatementToDatabaseException', false) === true) {
```

Comment:
Should we update the deprecationWarning below and have it recommend that folks enable this setting?

Reply:
Done.